pallets/statistics/src/benchmarking.rs

```diff
@@ -0,0 +1,123 @@
+use crate::*;
+use core::convert::TryFrom;
+use frame_benchmarking::benchmarks;
+use polymesh_common_utilities::{
+    asset::AssetType,
+    benchs::{User, UserBuilder},
+};
+use sp_std::prelude::*;
+
+/// Create a new token with name `name` on behalf of `owner`.
+/// The new token is a _divisible_ one with 1_000_000 units.
+pub fn make_token<T: Trait>(owner: &User<T>, name: Vec<u8>) -> Ticker {
+    let ticker = Ticker::try_from(name.as_slice()).unwrap();
+    T::Asset::create_asset(
+        owner.origin.clone().into(),
+        name.into(),
+        ticker.clone(),
+        1_000_000.into(),
+        true,
+        AssetType::default(),
+        Vec::new(),
+        None,
+    )
+    .expect("Cannot create an asset");
+
+    ticker
+}
+
+benchmarks! {
+    _ {}
+
+    add_transfer_manager {
+        let owner = UserBuilder::<T>::default().generate_did().build("OWNER");
+        let ticker = make_token::<T>(&owner, b"1".to_vec());
+        let mut tms = Vec::new();
+        for i in 1..T::MaxTransferManagersPerAsset::get() {
+            tms.push(TransferManager::CountTransferManager(i.into()));
+        }
+        ActiveTransferManagers::insert(ticker, tms.clone());
+        tms.push(TransferManager::CountTransferManager(420));
+    }: _(owner.origin, ticker, tms.last().unwrap().clone())
+    verify {
+        assert!(Module::<T>::transfer_managers(ticker) == tms);
+    }
+
+    remove_transfer_manager {
+        let owner = UserBuilder::<T>::default().generate_did().build("OWNER");
+        let ticker = make_token::<T>(&owner, b"1".to_vec());
+        let mut tms = Vec::new();
+        for i in 0..T::MaxTransferManagersPerAsset::get() {
+            tms.push(TransferManager::CountTransferManager(i.into()));
+        }
+        ActiveTransferManagers::insert(ticker, tms.clone());
+    }: _(owner.origin, ticker, tms.pop().unwrap().clone())
```

Comment:
Let's bind `tms.pop().unwrap().clone()` in the setup phase; this way you've written it here it will contribute to the bench weight. Same in previous bench.